hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/topic/TopicService.java

```diff
@@ -57,6 +57,15 @@ public void createTopic(Topic topic) {
         topicValidator.ensureCreatedTopicIsValid(topic);
         topicRepository.createTopic(topic);
 
+        if (!multiDCAwareService.topicExists(topic)) {
+            createTopicInBrokers(topic);
+        } else {
+            logger.info(String.format("Skipping creating topic %s on brokers, topic already exists", topic.getQualifiedName()));
```

Comment:
`logger.info("Skipping creation of topic {} on brokers, topic already exists", topic.getQualifiedName())`